src/server/cluster/cluster_config_test.cc

```diff
@@ -96,7 +96,7 @@ TEST_F(ClusterConfigTest, ConfigSetInvalidEmpty) {
 
 TEST_F(ClusterConfigTest, ConfigSetInvalidMissingSlots) {
   EXPECT_EQ(ClusterConfig::CreateFromConfig(
-                kMyId, {{.slot_ranges = {{.start = 0, .end = 16000}},
+                kMyId, {{.slot_ranges = std::vector<SlotRange>{{.start = 0, .end = 16000}},
```

Comment:
Any reason not to use `SlotRanges` in `ClusterConfig`?

Reply:
std::vector<SlotRange> is used to create SlotRanges, it is implicit creation of SlotRanges. I can do it explicit if you want

Reply:
I see. In that case, I think it would be clearer to use `SlotRanges({{.start=...}})`